packages/dos-donts-block/src/settings.ts

```diff
@@ -1,142 +1,464 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import { AssetInputSize, DropdownSize } from '@frontify/fondue';
 import {
-    DropdownSize,
+    AssetChooserObjectType,
+    FileExtension,
     IconEnum,
     appendUnit,
     defineSettings,
     numericalOrPixelRule,
     presetCustomValue,
 } from '@frontify/guideline-blocks-settings';
-import { DoDontLayout, DoDontSpacing, DoDontStyle, spacingValues } from './types';
+import { Radius, getBorderRadiusSettings, getBorderSettings, radiusStyleMap } from '@frontify/guideline-blocks-shared';
+import {
+    BlockMode,
+    DoDontImageHeight,
+    DoDontSpacing,
+    DoDontStyle,
+    DoDontVisibility,
+    GUTTER_VALUES,
+    IMAGE_HEIGHT_VALUES,
+    ImageFitChoice,
+    ItemIconChoice,
+} from './types';
 
 export const DO_COLOR_DEFAULT_VALUE = { red: 0, green: 200, blue: 165, alpha: 1 };
 export const DONT_COLOR_DEFAULT_VALUE = { red: 255, green: 55, blue: 90, alpha: 1 };
 
-const SPACING_VALUE_ID = 'spacingValue';
-const SPACING_CHOICE_ID = 'spacingChoice';
-
 export const settings = defineSettings({
     main: [
         {
-            id: 'style',
+            id: 'mode',
```

Comment:
Could you comment or remove `mode`, so we don't have it for the initial rollout of the text block?

Reply:
Sure, done!